src/converter/one/convert/JfrToOtlp.java

```diff
@@ -0,0 +1,267 @@
+/*
+ * Copyright The async-profiler authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package one.convert;
+
+import static one.convert.OtlpConstants.*;
+
+import one.jfr.JfrReader;
+import one.jfr.StackTrace;
+import one.jfr.event.Event;
+import one.proto.Proto;
+
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+/** Converts .jfr output to OpenTelemetry protocol. */
+public class JfrToOtlp extends JfrConverter {
```

Comment:
I see information about loaded native libraries is available via `jdk.NativeLibrary` events. Do we need to encode that information? It's not particularly useful unless we have the address for instructions in each sample frame.

Reply:
Library name is already encoded as class name in JFR Method structure (`MethodRef.cls`).